### ISSUE TITLE
Where is the beginning of that form?

### ISSUE THREAD

**Issue by alainrafiki** [RobRenfroe/google-homepage#1]
I see the closing of a form on line 16, but where is the beginning of it? Is it inside the script on line 14? I am referring to the form at the following line of the index file:
https://github.com/RobRenfroe/google-homepage/blob/37fe89e0691e1872db57f94982a2ab0e0aa35058/index.html#L16

**Comment by RobRenfroe** [comment]
Yes the script and form were all generated from google . I found through a
google search that I can generate a customer google search box and google
gave me the syntax for it
